src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -4,6 +4,7 @@
 import ai.susi.json.JsonObjectWithDefault;
 import ai.susi.mind.SusiSkill;
 import ai.susi.server.*;
+import com.sun.org.apache.xpath.internal.operations.Bool;
```

Comment:
This looks like it isn't being used at all. Could you please remove this import dependency?
Once this is done, I believe it's good to merge.